### PR TITLE
updates cache handler test to use strechr/testify to make each test runnable from IDE

### PR DESCRIPTION
**Pull request by ankitaluthra1**
### Link to the issue in case of a bug fix.
b/345645002

### Testing details
1. Manual - NA
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

File: internal/cache/file/cache_handle_test.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ import (
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/storageutil"
 	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/operations"
 	. "github.com/jacobsa/ogletest"
+	"github.com/stretchr/testify/suite"
```
Comment (Tulsishah) [inline internal/cache/file/cache_handle_test.go:40]: Aren't we removing full dependency from `github.com/jacobsa/ogletest` ?

Comment (ankitaluthra1) [reply internal/cache/file/cache_handle_test.go:40]: removed github.com/jacobsa/ogletest import

File: internal/cache/file/cache_handle_test.go [hunk 2]
```diff
@@ -113,16 +115,16 @@ func (cht *cacheHandleTest) SetUp(*TestInfo) {
...
 	testObjectContent := make([]byte, TestObjectSize)
 	n, err := rand.Read(testObjectContent)
-	AssertEq(TestObjectSize, n)
-	AssertEq(nil, err)
+	assert.Equal(cht.T(), TestObjectSize, n)
+	assert.Equal(cht.T(), nil, err)
 	objects := map[string][]byte{TestObjectName: testObjectContent}
 	err = storageutil.CreateObjects(ctx, cht.bucket, objects)
-	AssertEq(nil, err)
+	assert.Equal(cht.T(), nil, err)
```
Comment (Tulsishah) [inline internal/cache/file/cache_handle_test.go:122]: nit: assertNil
Consider it on other places also.

File: internal/cache/file/cache_handle_test.go [hunk 3]
```diff
@@ -211,7 +213,7 @@ func (cht *cacheHandleTest) Test_IsSequential_WhenPrevOffsetGreaterThanCurrent()
 
 	isSeq := cht.cacheHandle.IsSequential(currentOffset)
 
-	ExpectEq(false, isSeq)
+	assert.Equal(cht.T(), false, isSeq)
```
Comment (Tulsishah) [inline internal/cache/file/cache_handle_test.go:216]: nit: assertFalse?
and other places also.

**Review by Tulsishah — APPROVED** [review]
Approved with 2-3 nits.

File: internal/cache/file/cache_handle_test.go (hunk 3 above)
Comment (ankitaluthra1) [reply internal/cache/file/cache_handle_test.go]: done

File: internal/cache/file/cache_handle_test.go (hunk 2 above)
Comment (ankitaluthra1) [reply internal/cache/file/cache_handle_test.go]: done

File: internal/cache/file/cache_handle_test.go [hunk 4]
```diff
@@ -524,10 +526,10 @@ func (cht *cacheHandleTest) Test_Read_WithNilFileHandle() {
 
 	n, cacheHit, err := cht.cacheHandle.Read(context.Background(), cht.bucket, cht.object, offset, dst)
 
-	ExpectNe(nil, err)
-	ExpectEq(0, n)
-	ExpectFalse(cacheHit)
-	ExpectEq(util.InvalidFileHandleErrMsg, err.Error())
+	assert.NotEqual(cht.T(), nil, err)
```
Comment (Tulsishah) [inline internal/cache/file/cache_handle_test.go:529]: Sorry missed this.
nit: asset.NotNil()
Other places too

**Review by Tulsishah — APPROVED** [review]

File: internal/cache/file/cache_handle_test.go (hunk 4 above)
Comment (ankitaluthra1) [reply internal/cache/file/cache_handle_test.go]: Done

**Review by Tulsishah — DISMISSED** [review]

File: internal/cache/file/cache_handle_test.go [hunk 5]
```diff
@@ -46,9 +47,10 @@ const TestObjectSize = 16 * util.MiB
 const TestObjectName = "foo.txt"
 const DefaultSequentialReadSizeMb = 17
 
-func TestCacheHandle(t *testing.T) { RunTests(t) }
+//func TestCacheHandle(t *testing.T) { RunTests(t) }
```
Comment (ashmeenkaur) [inline internal/cache/file/cache_handle_test.go:50]: Nit: remove this line

Comment (ankitaluthra1) [reply internal/cache/file/cache_handle_test.go]: done

**Review by ashmeenkaur — APPROVED** [review]

**Pull request merged**